src/components/NewBeer.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react'
+import Header from './Header'
+
+export default class NewBeer extends Component {
+
+    state = {
+        name: "",
+        tagline: "",
+        description: "",
+        first_brewed: "",
+        brewers_tips: "",
+        attenuation_level: "",
+        contributed_by: ""
+    }
+
+    //how to define state type?
+
+    handleChange(event) {
+        let { name, value } = event.target;
+        this.setState({[name]: value});
+    }
+    //Do I need handle change?
+
+    handleFormSubmit = (event) => {
+        event.preventDefault();
+        console.log(this.state);
+
```

Comment:
`const {name, tagline, brewers_tips, first_brewed, attenuation_level, description, contributed_by} = this.state` You can use the array destructuring in order to use a shorter syntax on line 36-42 (you don't have to but in some cases it makes your code more readable and easier to write)